### PR TITLE
All claims 32 mg

### PR DESCRIPTION
**Pull request by markgreenburg**
~*NB:* Requires [this change](https://github.com/department-of-veterans-affairs/vets-website/pull/8475) to be merged into `vets-website` before this is merged.~ merged

## Description
- Adds claim exams info page to 526 All Claims Form
- Updates Formation dependency to pull in changes that address bug with form continuing when collapsible panels toggled

## Testing done
- Tested locally

## Screenshots

Panels start collapsed by default:
screen shot 2018-09-20 at 1 58 10 pm

-----

Clicking on a panel expands it:
screen shot 2018-09-20 at 2 23 03 pm

### CODE REVIEW

**Review by cvalarida — APPROVED** [review]

**Comment by markgreenburg** [comment]
Confirmed with Alex that the intended UI here is the accordion.

File: src/applications/disability-benefits/all-claims/content/claimExamsInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  </p>
+);
+
+export const claimExamsFAQ = (
+  <div>
+    <CollapsiblePanel panelName="Does everyone who files a claim need to have a VA claim exam?">
+      <p>
+        No. We’ll ask you to have a claim exam only if we need more information
+        to decide your claim. If you need to have an exam, you should receive
+        your first call within XX days.
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/content/claimExamsInfo.jsx]: Discussing with Peggy

File: src/applications/disability-benefits/all-claims/content/claimExamsInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      </p>
+    </CollapsiblePanel>
+    <CollapsiblePanel panelName="How we schedule exams">
+      <p>
+        The staff at your local VA medical center will contact you to schedule your
+        exam. An exam scheduler will make two attempts to contact you by phone to
+        find a time that works for you. The phone number they’ll call you from will
+        look like this:<br/>
+        NNN-NNN-XXXX<br/>
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/content/claimExamsInfo.jsx]: Checking with Peggy to see if this is correct.

**Review by peggygannon — CHANGES_REQUESTED** [review]
@markgreenburg Please see below for some updates to the outline. It was helpful to see this content laid out and see where I need to adjust, esp with the accordions.
I removed the toggle answer from the first question, as this question/answer is important and works better untoggled. (not everyone will need an exam). 

Please let me know if you have any question about my updates below. Thank you!

After we review your disability claim and supporting evidence, we may ask you to have a claim exam (also known as a C&P exam) if we need more information to decide your claim. 
We’ll use the results of this exam, in addition to your evidence, to rate your disability and make a decision on your claim. 

**Does everyone who files a disability claim need to have a claim exam?** 
No. We’ll ask you to have a claim exam only if we need more information to decide your claim. If you've provided enough supporting evidence for your claimed disability, we won’t ask you to have a claim exam. 

**How will you contact me if I need to have an exam?**
{Toggle answer}
The staff at your local VA medical center will contact you to schedule your exam. They’ll either send you a letter by mail with the date and time of your exam, or call you to find a time that works for you. 

**What if I need to reschedule my exam appointment?**
{Toggle answer}
If you can’t make it your scheduled exam, let us know right away. You can most likely reschedule, but this may delay a decision on your claim. If you need to reschedule your exam, please call 1-800-827-1000.

[ Back | Continue ] { buttons }
Save and finish this application later.

**Comment by markgreenburg** [comment]
@peggygannon please see new screenshots per our convo 🙂

**Review by peggygannon — APPROVED** [review]

**Comment by peggygannon** [comment]
thanks, @markgreenburg Updated screenshots look good.

**Pull request merged**